### ISSUE TITLE
Users should be able to set a custom configuration

### ISSUE THREAD

**Issue by landakram** [landakram/kiwi#5]
This would include things like hiding / showing the status bar, link behavior, toolbar options, etc.

**Comment by mry53** [comment]
Custom configuration to include edit mode enhancements? I'm especially thinking of setting margins; current full-screen width margins are extremely hard to use. Text is easier to write, read, and parse when column widths are narrower than full-screen width

**Comment by landakram** [comment]
@mry53 that's definitely a good candidate for configuration! Are you primarily working on an iPad where the edit screen is much wider?

**Comment by mry53** [comment]
Yes, I am. It's a use case that too many editors don't get right (in which case they don't get used, or recommended...)

from my iPad

**Comment by matiasvillaverde** [comment]
@landakram is possible to configure the links syntax as well? Because I am using Github wiki or Gollum where the links of a file "My-Page.md" is "[[My Page]]". While Kiwi's links are [[My-Page]]. Thanks in advance!
